slushfile.js

```diff
@@ -233,6 +238,18 @@ function configRoxy() {
       '          <range-value-positions>false</range-value-positions>\n' +
       '        </range-element-index>\n');
 
+    // fix default app-role privileges to match rest-style applications
+    foo = foo.replace(/<privileges>[^]*?<\/privileges>/,
+      '<privileges>\n' +
+      '        <privilege>\n' +
+      '          <privilege-name>rest-reader</privilege-name>\n' +
+      '        </privilege>\n' +
+      '        <!-- remove the rest-writer privilege if read-only access is required -->\n' +
+      '        <privilege>\n' +
+      '          <privilege-name>rest-writer</privilege-name>\n' +
+      '        </privilege>\n' +
+      '      </privileges>\n');
+
```

Comment:
Decided to include a pre-made app_specific.rb in templates to solve this. Showcases some other nice gimmicks as well..